ext/web/06_streams.js

```diff
@@ -4734,6 +4757,32 @@ const asyncIteratorPrototype = ObjectGetPrototypeOf(AsyncGeneratorPrototype);
 const _iteratorNext = Symbol("[[iteratorNext]]");
 const _iteratorFinished = Symbol("[[iteratorFinished]]");
 
+class ReadableStreamAsyncIteratorReadRequest {
+  constructor(reader, promise) {
+    this.reader = reader;
```

Comment:
This is vulnerable if a user declares an object getter/setter. Explicitly define the property on the class (maybe private is even faster?)

Reply:
Agree with making them private, just pushed it. But how is it vulnerable? The class is not exposed and only uses internal fields.

Reply:
If a user adds a `stream` setter/getter to `Object.prototype`, this assignment will call that setter rather than creating an object property on this.

Reply:
To fix it, just declaring the field on the class is enough (`class X { stream; constructor() { ... } }`). This causes the locally declared field to shadow the possible setter/getter on the object prototype